tile/TileSeparateCrossbarRTL.py

```diff
@@ -186,23 +186,44 @@ def update_opt():
       s.routing_crossbar.recv_opt.msg @= s.ctrl_mem.send_ctrl.msg
       s.fu_crossbar.recv_opt.msg @= s.ctrl_mem.send_ctrl.msg
 
-      s.element.recv_opt.en @= s.ctrl_mem.send_ctrl.en
-      s.routing_crossbar.recv_opt.en @= s.ctrl_mem.send_ctrl.en
-      s.fu_crossbar.recv_opt.en @= s.ctrl_mem.send_ctrl.en
-
-      s.ctrl_mem.send_ctrl.rdy @= s.element.recv_opt.rdy & \
-                                  s.routing_crossbar.recv_opt.rdy & \
-                                  s.fu_crossbar.recv_opt.rdy
-
+      s.element.recv_opt.val @= s.ctrl_mem.send_ctrl.val & ~s.element_done
+      s.routing_crossbar.recv_opt.val @= s.ctrl_mem.send_ctrl.val & ~s.routing_crossbar_done
+      s.fu_crossbar.recv_opt.val @= s.ctrl_mem.send_ctrl.val & ~s.fu_crossbar_done
+
+      # s.ctrl_mem.send_ctrl.rdy @= s.element.recv_opt.rdy & \
+      #                             s.routing_crossbar.recv_opt.rdy & \
+      #                             s.fu_crossbar.recv_opt.rdy
+
+      # FIXME: yo96, rename ctrl.rdy to ctrl.proceed or sth similar.

```

Comment:
@yo96 Leave a comment for you to rename the signal about the ctrl-related signals (i.e., not real val/rdy, but simple en or proceed or incr).